cinema/permissions.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework.permissions import BasePermission, SAFE_METHODS
```

Comment:
There is a syntax error in the import statement. The slash (`/`) should be replaced with a comma (`,`) to correctly import `SAFE_METHODS` from `rest_framework.permissions`.